api/controllers/comment.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+
+const config = require('../config');
+const db = require('../models');
+const common = require('../common');
+
+
+/**
+ * @apiGroup Comment
+ * @api {get} /comment/project/:projectid  Get Comment List of Particular Project
+ * @apiHeader {String} authorization 
+ *                              A valid JWT token "Bearer: xxxxx"
+ * @apiSuccess {Object}            List of Comments 
+ */
+
+router.get('/project/:id', common.jwt(), (req, res, next) =>{
+    if(!req.params.id) return next("project id not defined");
```

Comment:
I don't thinks this check is necessary as this route will never match without `params.id` set.